_docs/samples/bookinfo.md

```diff
@@ -10,8 +10,17 @@ layout: docs
 type: markdown
 ---
 
+This sample deploys a simple application composed of four separate microservices which will be used
+to demonstrate various features of the Istio service mesh.
+          
+## Before you begin
+
+Setup the Istio controlplane services by following the instructions in the
```

Comment:
We're trying to avoid 'control plane' since it is very vague and confusing people. So how about just "Setup Istio by following..."